gateway/src/resty/http_ng/headers.lua

```diff
@@ -54,7 +56,7 @@ local header_mt = {
   __tostring = function(t)
     local tmp = {}
 
-    for k,v in pairs(t) do
+    for k,v in tablex_sort(t) do
```

Comment:
Why do headers need to be sorted? Is it to pass some tests or is there some part of the code that expects them sorted?

Reply:
Just to pass the test, is the only way to have consistent data.